### ISSUE TITLE
haxball bot 

### ISSUE THREAD

**Issue by disneyy** [haxball/haxball-issues#886]
Hello there,
I want administrators to write in color, but I can't. Can you help me ? thank u

**Comment by Wazarr94** [comment]
Hello, not possible, so far only headless bot can use the personalized fonts and colors

**Comment by saviola777** [comment]
Well, he could intercept all messages from administrators and re-send them through `sendAnnouncement`.

**Comment by Wazarr94** [comment]
Ah yes, in my head I was thinking about non headless rooms.

**Comment by disneyy** [comment]
yes only headless bot can write colorful but if player_admin keep the message and send it in color

**Comment by disneyy** [comment]
> Hello, not possible, so far only headless bot can use the personalized fonts and colors

Yes only headless bot can write colorful but bot can store messages and resend in color

**Comment by mob-hash** [comment]
This is so simple actually. The method @saviola777 mentioned is what you need.
